reference/specs/bindings/rabbitmq.md

```diff
@@ -16,11 +16,14 @@ spec:
     value: true
   - name: deleteWhenUnused
     value: false
+  - name: ttlInSeconds
+    value: 60
 ```
 
 - `queueName` is the RabbitMQ queue name.
 - `host` is the RabbitMQ host address.
 - `durable` tells RabbitMQ to persist message in storage.
 - `deleteWhenUnused` enables or disables auto-delete.
+- `ttlInSeconds` is a optional paramater to set the [default message time to live at RabbitMQ queue level](https://www.rabbitmq.com/ttl.html). If this option is omitted, messages won't expire, existing until processed.
```

Comment:
Please add a sample request showing the metadata as done here for all the bindings doc(RabbitMQ. Service Bus, Service Queue):
https://github.com/dapr/docs/blob/master/reference/specs/bindings/kafka.md#specifying-a-partition-key